modules/programs/mcfly.nix

```diff
@@ -30,8 +30,19 @@ in {
     enableFuzzySearch = mkOption {
```

Comment:
You should remove this option entirely. Instead add

```nix
imports = [
  (mkChangedOptionModule # \
    [ "programs" "mcfly" "enableFuzzySearch" ] # \
    [ "programs" "mcfly" "fuzzySearchFactor" ] (config:
      let
        value =
          getAttrFromPath [ "programs" "mcfly" "enableFuzzySearch" ] config;
      in if value then 2 else 0))
];
```

after the `meta.maintainers = [ maintainers.marsam ];` line.